pkg-plist

```diff
@@ -3,6 +3,7 @@ pkg/birdvector.inc
 bin/pathvector
 /etc/inc/priv/birdvector.priv.inc
 www/packages/birdvector/birdvector.php
+www/packages/birdvector/birdvector_about.php
```

Comment:
Need to also add the new `birdvector_about.php` file into the `Makefile` and also into `files/etc/inc/priv/birdvector.priv.inc`. Dont ask me why, this is the 'FreeBSD way...'